dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -260,6 +260,18 @@ public List<String> listFiles(String repositoryId, String pathToDirectory, Strin
         }
     }
 
+    public byte[] readZip(String repositoryId, String reference) {
```

Comment:
Should javadoc public method in often-used class

Reply:
There may be a plugin for emacs/vim, to get automatic placeholders for `@param` and `@return` and they are technically required for Javadoc, otherwise this is just a regular comment. 

https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#requiredtags